packages/kbn-router-to-openapispec/src/process_router.ts

```diff
@@ -63,10 +147,16 @@ export const processRouter = (
         parameters.push(...pathObjects, ...queryObjects);
       }
 
+      const authzDescription = extractAuthzDescription(route, buildFlavour);
+
+      const description = `${route.options.description ?? ''}${authzDescription ?? ''}`;
```

Comment:
Perhaps to help the layout a bit 🤷🏻 
```suggestion
      const description = `${route.options.description ?? ''}${route.options.description?.length ? ' ' : '' }${authzDescription ?? ''}`;
```